src/main/fc/fc_msp_box.c

```diff
@@ -76,7 +76,7 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOXCAMERA1, "CAMERA CONTROL 1", 32},
     { BOXCAMERA2, "CAMERA CONTROL 2", 33},
     { BOXCAMERA3, "CAMERA CONTROL 3", 34 },
-    { BOXDSHOTREVERSE, "DSHOT REVERSE MOTORS", 35 },
+    { BOXDSHOTTURTLE, "TURTLE (DSHOT ONLY)", 35 },
```

Comment:
This is utterly confusing to somebody who has not been following the communication. Can we rename this to something that actually indicates the function, and not just a buzzword?

Reply:
Open to suggestions.  Reverse motors didn't seem to apply any more and didn't really make sense for righting after a crash.  Talking with people turtle mode seemed the term that most people use to describe what is going on.   Me: you can reverse the motors after a crash to flip.  Them: Oh you mean turtle mode.

We aren't really buzzword free with other modes as well.  

I wanted the reverse motors (or similar name) free to implement 3d mode on a stick.

Reply:
Fair enough about the '3D on a stick'. How about using 'Flip' for flipping the quad over after a crash?

Reply:
Thought about that, but sounds like the auto-flip mode on old brushed quads.  Also doesn't automatically flip you over and still need stick input to cause it to flip over.

Reply:
Well, it *allows you* to flip. I don't think auto-flip was ever a thing in Betaflight, so there is little chance of confusing the two.